src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -272,8 +267,7 @@ export const ChannelInput: React.FC<IChannelInput> = ({
     return splitedMsg.join(String.fromCharCode(160))
   }
 
-  const t2 = sanitizeHtml(htmlMessage)
-  const sanitizedHtml = findMentions(t2)
+  const sanitizedHtml = findMentions(message)
```

Comment:
I suspect something is wrong here..

`sanitizedHtml` is given few lines below to:
```
                <ContentEditable
                     ....
                     html={sanitizedHtml}
```

Will it be fine with us no longer calling `sanitizeHtml` ?